keyring/keyring.go

```diff
@@ -0,0 +1,118 @@
+package keyring
+
+import (
+	"crypto/ed25519"
+	"crypto/rand"
+	"encoding/base64"
+	"os"
+	"path/filepath"
+
+	"github.com/deislabs/go-bindle/types"
+	"github.com/pelletier/go-toml"
+)
+
+// GenerateSigningKey generates a keypair for signing Bindle invoices
```

Comment:
Looks like you'll need to update the function name here so it counts as the doc comment